src/queries/tags.ts

```diff
@@ -2,6 +2,8 @@ export enum QueryTagTypes {
   Funders = 'Funders',
   FundingEntities = 'FundingEntities',
   UserFundingEntity = 'UserFundingEntity',
+  Observation = 'Observation',
+  ObservationMedia = 'ObservationMedia',
```

Comment:
Probably want to keep these alphabetical in this file, so they should be moved above `UserFundingEntity`. No worries if they're just appended like this as part of the code gen.